extensions/configuration/oauth2.yaml

```diff
@@ -206,3 +254,11 @@ data:
         spec.clientName: Client Name
         secret.client_id: client_id
         secret.client_secret: client_secret
+        deprecation.title: Ory Hydra Deprecation on May 19, 2022
+        deprecation.long: |
+          Dear customer,
+          you are using Ory Hydra OAuth2, which has been deprecated since May 19, 2022 and is scheduled to be removed on October 14, 2023.
```

Comment:
```suggestion
          You are using Ory Hydra OAuth2, which has been deprecated since May 19, 2022 and is scheduled to be removed on October 14, 2023.
```